src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -117,44 +99,56 @@ const SuggestedOwners = createReactClass({
     return (
       <span
         key={author.id || author.email}
-        className="avatar-grid-item tip"
+        className="avatar-grid-item"
         onClick={() => this.assignTo(author)}
-        title={ReactDOMServer.renderToStaticMarkup(
-          <div>
-            {author.id ? (
-              <div className="tooltip-owners-name">{author.name}</div>
-            ) : (
-              <div className="tooltip-owners-unknown">
-                <p className="tooltip-owners-unknown-email">
-                  <span className="icon icon-circle-cross" />
-                  <strong>{author.email}</strong>
-                </p>
-                <p>
-                  Sorry, we don't recognize this member. Make sure to link alternative
-                  emails in Account Settings.
-                </p>
-                <hr />
-              </div>
-            )}
-            <ul className="tooltip-owners-commits">
-              {commits.slice(0, 6).map(c => {
-                return (
-                  <li key={c.id} className="tooltip-owners-commit">
-                    <div style={{whiteSpace: 'pre-line'}}>
-                      {c.message.replace(/\n\s*\n/g, '\n') /*repress repeated newlines*/}
-                    </div>
-                    <span className="tooltip-owners-date">
-                      {' '}
-                      - {moment(c.dateCreated).fromNow()}
-                    </span>
-                  </li>
-                );
-              })}
-            </ul>
-          </div>
-        )}
       >
-        <Avatar user={author} />
+        <Tooltip
+          tooltipOptions={{
+            html: true,
+            container: 'body',
+            template:
+              '<div class="tooltip" role="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner tooltip-owners"></div></div>',
+          }}
+          title={ReactDOMServer.renderToStaticMarkup(
```

Comment:
Will this work as a react node?

Reply:
sadly not as-is with bootstrap :/ we could make an an adapter inside the component